w5500-lwIP.cpp

```diff
@@ -61,17 +82,39 @@ err_t Wiznet5500lwIP::linkoutput_s (netif *netif, struct pbuf *pbuf)
 {
     Wiznet5500lwIP* ths = (Wiznet5500lwIP*)netif->state;
     
+    /*
     if (pbuf->len != pbuf->tot_len || pbuf->next)
         Serial.println("ERRTOT\r\n");
 
     uint16_t len = ths->sendFrame((const uint8_t*)pbuf->payload, pbuf->len);
+    */
+
+    struct pbuf *p = pbuf;
+
+    //if (p->len != p->tot_len)
+    //  Serial.printf("Chained : p->len=%d p->tot_len=%d\r\n", p->len, p->tot_len);
+
+    do 
```

Comment:
I think chained pbufs must be unchained prior to sending the full frame.
A fragmented pbuf in this function is a continous frame (ethernet packet) splitted into parts.
sendFrame expects a valid monolithic ethernet packet.